gammapy/visualization/datasets.py

```diff
@@ -0,0 +1,175 @@
+import matplotlib.pyplot as plt
+
+__all__ = [
+    "plot_spectrum_datasets_off_regions",
+    "plot_npred_signal",
+]
+
+
+def plot_spectrum_datasets_off_regions(
+    datasets, ax=None, legend=None, legend_kwargs=None, **kwargs
+):
+    """Plot the off regions of spectrum datasets.
+
+    Parameters
+    ----------
+    datasets : `~gammapy.datasets.Datasets` or list of `~gammapy.datasets.SpectrumDatasetOnOff`
```

Comment:
Is `~gammapy.datasets.Datasets` apt here? Only `SpectrumDatasetOnOff`s are supported, so `list of ~gammapy.datasets.SpectrumDatasetOnOff` should suffice.